README.md

```diff
@@ -1,21 +1,94 @@
-[![Build Status](https://travis-ci.org/awslabs/serverless-application-model.svg?branch=develop)](https://travis-ci.org/awslabs/serverless-application-model)
-[![PyPI version](https://badge.fury.io/py/aws-sam-translator.svg)](https://badge.fury.io/py/aws-sam-translator)
-[![Codecov Test Coverage](https://codecov.io/gh/awslabs/serverless-application-model/branch/master/graphs/badge.svg?style=flat)](https://codecov.io/gh/awslabs/serverless-application-model)
-
-![Logo](aws_sam_introduction.png)
+<p align="center">
+</p>
 
 # AWS Serverless Application Model (AWS SAM)
-You can use SAM to define serverless applications in simple and clean syntax.
 
-This GitHub project is the starting point for AWS SAM. It contains the SAM specification, the code that translates SAM templates into AWS CloudFormation stacks, general information about the model, and examples of common applications.
+![Apache-2.0](https://img.shields.io/github/license/awslabs/serverless-application-model.svg)
+![SAM_CLI release](https://img.shields.io/github/release/awslabs/aws-sam-cli.svg)
```

Comment:
Why is the SAM CLI release version here?

Reply:
I wanted to add badges for information that customers commonly interact with - it is usually the CLI version.